src/AutoRest.CSharp/Common/Generation/Writers/CodeWriter.cs

```diff
@@ -155,6 +156,96 @@ public void UseNamespace(string @namespace)
             }
         }
 
+        public CodeWriter AppendXmlDocumentation(FormattableString startTag, FormattableString endTag, FormattableString content)
+        {
+            const string xmlDoc = "/// ";
+            const string xmlDocNewLine = "\n/// ";
+
+            var commentStart = _length;
+            AppendRaw(CurrentLine.IsEmpty ? xmlDoc : xmlDocNewLine);
+
+            var startTagStart = _length;
+            Append(startTag);
+
+            var contentStart = _length;
+            if (content.Format.Length > 0)
+            {
+                Append(content);
+            }
+            var contentEnd = _length;
+
+            Append(endTag);
+
+            if (contentStart == contentEnd)
+            {
+                var startTagSpan = WrittenText.Slice(startTagStart + 1, contentStart - startTagStart - 1);
+                var endTagSpan = WrittenText.Slice(contentEnd + 2);
+
+                if (startTagSpan.SequenceEqual(endTagSpan))
+                {
+                    // Remove empty tags
+                    _length = commentStart;
+                }
+                else
+                {
+                    // Insert " . " to match existing formatting
+                    InsertRaw(" . ", contentEnd);
```

Comment:
This is needed only to avoid hundreds of changes that remove ` . ` from `parameter` tag.

Reply:
I'd rather we re-gen what's required instead of carrying this complex logic forward.

Reply:
Fixed. +250 changed files :)